src/main/java/ua/lysenko/andrii/zip/web/config/StorageConfiguration.java

```diff
@@ -0,0 +1,34 @@
+package ua.lysenko.andrii.zip.web.config;
+
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import ua.lysenko.andrii.zip.web.FileSystemService;
+import ua.lysenko.andrii.zip.web.StorageService;
+
+import java.nio.file.Paths;
+
+@Configuration
```

Comment:
This annotation is needed to use bean initialization via method `storageService`, isn't it?